### PR TITLE
update cuda version in unittest_windows_gpu

### PR DESCRIPTION
**Pull request by mszhanyi**
To pass the test_decode_jpeg_cuda, the cuda version should be 11.1
image
https://github.com/pytorch/vision/pull/4421#issuecomment-926021032

else, it would throw an exception.
image

The change is verified.
https://app.circleci.com/pipelines/github/pytorch/vision/10657/workflows/b22cfed7-4910-47ea-ac7e-8cf40adaf646/jobs/811597

### CODE REVIEW

File: .circleci/unittest/windows/scripts/install.sh [hunk 1]
```diff
@@ -26,7 +26,8 @@ else
 fi
 
 printf "Installing PyTorch with %s\n" "${cudatoolkit}"
-conda install -y -c "pytorch-${UPLOAD_CHANNEL}" "pytorch-${UPLOAD_CHANNEL}"::pytorch "${cudatoolkit}" pytest
+# why add -c conda-forge again, see https://github.com/pytorch/vision/issues/4458
+conda install -y -c "pytorch-${UPLOAD_CHANNEL}" -c conda-forge "pytorch-${UPLOAD_CHANNEL}"::pytorch "${cudatoolkit}" pytest
```
Comment (prabhat00155) [inline .circleci/unittest/windows/scripts/install.sh:30]: It's still unclear to me why are you adding conda-forge channel here?

**Review by prabhat00155 — CHANGES_REQUESTED** [review]

File: .circleci/unittest/windows/scripts/install.sh (hunk 1 above)
Comment (mszhanyi) [reply .circleci/unittest/windows/scripts/install.sh:30]: @prabhat00155 
without conda-forge, CI would throw exception that torch+cu11.1 isn't in that repo.

Comment (mszhanyi) [reply .circleci/unittest/windows/scripts/install.sh:30]: This link was used. I don't understand why there's a commit to remove all conda-forge. It's a right way to install torch+cu11.1 in CI.

Comment (mszhanyi) [reply .circleci/unittest/windows/scripts/install.sh:30]: https://app.circleci.com/pipelines/github/pytorch/vision/10705/workflows/25d6740c-6c8a-42fd-8cbf-7a54222ebb38/jobs/816118

```
Installing PyTorch with cudatoolkit=11.1
Collecting package metadata (current_repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - cudatoolkit=11.1

Current channels:

  - https://conda.anaconda.org/pytorch-nightly/win-64
  - https://conda.anaconda.org/pytorch-nightly/noarch
  - https://repo.anaconda.com/pkgs/main/win-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/win-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/msys2/win-64
  - https://repo.anaconda.com/pkgs/msys2/noarch
```

Comment (prabhat00155) [reply .circleci/unittest/windows/scripts/install.sh:30]: Okay, we need `conda-forge` on Windows for cu 11.1. Could you please update the comment you have on line 29 saying that: "`conda-forge` channel is required for cudatoolkit 11.1 on Windows."?

Comment (mszhanyi) [reply .circleci/unittest/windows/scripts/install.sh:30]: Sure, updated!

**Review by prabhat00155 — APPROVED** [review]
Looks good to me, thanks @mszhanyi!

**Pull request merged**